src/transformers/pipelines/conversational.py

```diff
@@ -196,9 +196,7 @@ def new_user_input(self):
     build_pipeline_init_args(has_tokenizer=True),
     r"""
         min_length_for_response (`int`, *optional*, defaults to 32):
-            The minimum length (in number of tokens) for a response.
-        minimum_tokens (`int`, *optional*, defaults to 10):
```

Comment:
`minimum_tokens` is an unused internal variable, probably a legacy version of `min_length`. 

Initially, I removed it from the signature of the private `_forward`. Then, I realized we can remove all traces, since it is unused :)